app/frontend/components/pages/delegations/currentDelegationPage.tsx

```diff
@@ -17,21 +17,25 @@ const CurrentDelegationPage = ({
           <div className="current-delegation-wrapper">
             {/* <div className="delegation-history-time">
               {formatDelegationDate(currentDelegation[0].time)}
-            </div>
+            </div>]
```

Comment:
why wrapping the commented section in "{}" and also this "]" does not seem to have a matching "["